### ISSUE TITLE
[1.12.2-5.2.0Alpha1] Crash when editing hardcore questing book

### ISSUE THREAD

**Issue by EMatzke1** [lorddusk/HQM#386]
I was editing the hardcore questing book for my custom pack when it crashed when I decided to delete something.

It came up with this crashlong: 
https://hastebin.com/nopavobori.sql

**Comment by canitzp** [comment]
I'm sorry that I have to tell you that we don't maintain the old codebase and you have to wait for the rewrite version

**Comment by canitzp** [comment]
@CrossJaw could you please reuplod the crash report

**Comment by EMatzke1** [comment]
I'm afraid I may have deleted it in the process of doing some late spring cleaning on my pc. I'm sorry for any inconveniences that this may cause. I can however remember what caused the crash in more detail than I had previously stated.

What I basically did is instead of crashing when I deleted anything at all, it was only when I deleted things that shouldn't meant to be deleted such as the symbol to go back to a previous page or the exit symbol to exit the book's interface and stuff like that. Things that are meant to be deleted (or at least what I thought) worked just fine. I'm sorry if I was non-descriptive the first time, but I hope this better description will help you even if I may not be able to provide the original crash-log.
